lib/tasks/admin.rake

```diff
@@ -228,7 +228,8 @@ namespace :admin do
         force_password_change: false,
         api_enabled: true,
         role: 'public_health_enroller',
-        is_api_proxy: true
+        is_api_proxy: true,
+        notes: ''
```

Comment:
I think this probably isn't necessary especially if the default value in the migration is already defined as ''